api/utils/requestProcessor.js

```diff
@@ -1408,6 +1408,12 @@ const processRequest = (params) => {
                         }
                         params.qstring.query.subtask = {$exists: false};
                         params.qstring.query.app_id = params.qstring.app_id;
+						if(params.qstring.app_ids && params.qstring.app_ids!=""){
```

Comment:
Codacy found an issue: [Expected '!==' and instead saw '!='.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6386294)